clang/include/clang/Driver/Driver.h

```diff
@@ -658,6 +661,19 @@ class Driver {
   static void getDefaultModuleCachePath(SmallVectorImpl<char> &Result);
 
   bool getOffloadStaticLibSeen() const { return OffloadStaticLibSeen; };
+
+  /// addFPGATempDepFile - Add a file to be added to the bundling step of
+  /// an FPGA object.
+  void addFPGATempDepFile(const std::string &Name,
+                          const std::string &JA) const {
+    std::pair<std::string, std::string> TP(JA, Name);
+    FPGATempDepFiles.insert(TP);
```

Comment:
```suggestion
    FPGATempDepFiles.insert({JA, Name});
```
(disregard-able in case of readability preferences)

Reply:
readable, IMO.  Done.